hstream/app/client.hs

```diff
@@ -123,17 +122,6 @@ hstreamNodes connOpts (HStreamNodesStatus mid) = do
       case serverNodeStatusNode of
         Just API.ServerNode{..} -> serverNodeId == x
         Nothing                 -> False
-hstreamNodes connOpts (HStreamNodesCheck nMaybe) = do
-  nodes <- describeClusterResponseServerNodesStatus <$> getNodes connOpts
-  let n' = length nodes
-  case nMaybe of
-    Just n -> when (n' < fromIntegral n) $ errorWithoutStackTrace "No enough nodes in the cluster"
-    Nothing -> return ()
-  let nodesNotRunning = V.filter ((/= EnumPB API.NodeStateRunning) . API.serverNodeStatusState) nodes
-  if null nodesNotRunning
-    then putStrLn "All nodes in the cluster are running."
-    else errorWithoutStackTrace $ "Some Nodes are not running: "
-                                <> show (mapMaybe ((API.serverNodeId <$>) . API.serverNodeStatusNode) (V.toList nodesNotRunning))
```

Comment:
Why delete this?

Reply:
Most of the functions of check running are overridden by node status, and the so-called check does not further examine the specific running status of the node, but is actually just a filter for the node status command